ast/misc.go

```diff
@@ -2001,7 +2001,7 @@ func (n *TableOptimizerHint) Restore(ctx *RestoreCtx) error {
 	switch n.HintName.L {
 	case "max_execution_time":
 		ctx.WritePlainf("%d", n.MaxExecutionTime)
-	case "tidb_hj", "tidb_smj", "tidb_inlj":
+	case "tidb_hj", "tidb_smj", "tidb_inlj", "access_from_flash":
```

Comment:
I think this hint should be a table hint?

Reply:
What do you mean? It exactly a table hint.